### ISSUE TITLE
Latex error for page with link target and without sections

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#611]
A sphinx-quickstart empty project with just the following index.rst:

```
Some text and alink_

.. _alink: http://www.python.org
```

will, on 'make latex', cause this error:

```
Exception occurred:
  File "/Users/mb312/usr/local/lib/python2.6/site-packages/Sphinx-1.1pre_60813183c80adev_20110117-py2.6.egg/sphinx/writers/latex.py", line 1029, in visit_target
    next = node.parent.parent[node.parent.parent.index(node.parent)]
AttributeError: 'NoneType' object has no attribute 'index'
```

Debug shows that the 'node' at fault is the link target, and it's expecting two levels above it in the document tree hierarchy.

Adding a section title to index.rst fixes it:

```
A title
-------

Some text and alink_

.. _alink: http://www.python.org
```

The same error results, if a more usual index.rst references a sectionless document like that above, i.e. where index.rst looks like:

```
Welcome to test's documentation!
================================

.. toctree::

   afile
```

and afile.rst is as above:

```
Some text and alink_

.. _alink: http://www.python.org
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/611
- Originally reported by: Matthew Brett
- Originally created at: 2011-01-18T01:20:43.856

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-21 08:36:35+00:00_

Fix #611: latex writer raised IndexError on pages with no section but a link target.

→ <<cset a4b8b81712fd080f2b25993add464c69ba6f1f39>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-21 08:47:51+00:00_

Fix #611: latex writer raised IndexError on pages with no section but a link target.

→ <<cset a4b8b81712fd080f2b25993add464c69ba6f1f39>>
